apps/ensadmin/src/components/recent-registrations/components.tsx

```diff
@@ -182,3 +188,82 @@ function RegistrationsListLoading({ rowCount }: RegistrationLoadingProps) {
     </div>
   );
 }
+
+function RecentRegistrationsLoading({ rowCount }: RegistrationLoadingProps) {
+  return (
+    <Card>
+      <CardHeader>
+        <CardTitle className="flex justify-between items-center">
+          <span>Latest indexed registrations</span>
+        </CardTitle>
+      </CardHeader>
+      <CardContent className="max-sm:p-3 max-sm:pt-0">
+        <RegistrationsListLoading rowCount={rowCount} />
```

Comment:
```suggestion
        <RegistrationsListLoading recordCount={maxRecords} />
```

In other words, the name of the param on `RegistrationsListLoading` should change from `rowCount` to `recordCount` for consistency. Note that this is NOT the same as `maxRecords` because in the context of a loading state it doesn't make any sense to talk about "max". You give the exact number of records you want in the loading state.